tests/foreman/cli/test_activationkey.py

```diff
@@ -183,7 +183,7 @@ def test_positive_create_with_usage_limit_default_explicitly(self):
 
         @Assert: Activation key is created
         """
-        new_ak = self._make_activation_key({u'unlimited-content-hosts': '1'})
+        new_ak = self._make_activation_key()
```

Comment:
Pls, read BZ description. It is not correct behaviour anyway and I know that information that you provided (sorry for my brief explanation).
Okay, I will remove the test entirely. Thanks